app/assets/javascripts/nbp_wallet.coffee.erb

```diff
@@ -0,0 +1,25 @@
+# subtracting 5 seconds to make sure the displayed code is always valid (accounting for loading times)
+template_validity = <%= (Enmeshed::ConnectorApi::TEMPLATE_VALIDITY_PERIOD).seconds - 5 %>
```

Comment:
When we would read this value through a data-attribute of the page, we would avoid changing this file to an ERB template. While it works, there is some caching issue with Sprockets, that we could only fix by adding a `depend_on` meta-comment (see https://github.com/openHPI/codeocean/pull/2288/commits/d62392db7074ff71151f1d3747ffc10cb461b91b for a similar example). Nothing too bad, just something to learn.

Reply:
Ok, I am not sure this works now but I also don't know how we would test that?

Reply:
Perfect, I've tested your change and can confirm that it works as expected (even when changing the validity value). Thus, I am closing this thread. 

Since you are now reading the data attribute directly, you also don't need the comment in `app/views/users/nbp_wallet/connect.html.slim`. This comment would only be needed in the CoffeeScript, if you would still use the `Enmeshed::ConnectorApi::TEMPLATE_VALIDITY_PERIOD` value.